src/Console/Commands/InstallAssets.php

```diff
@@ -0,0 +1,133 @@
+
+        $this->updateNodePackages(function ($packages) {
+            return data_get(
+                json_decode(
+                    file_get_contents(__DIR__ . '/../../../stubs/tailwind/package.json'),
+                    true
+                ),
+                'devDependencies'
+            ) + $packages;
+        });
+
+        if (file_exists(resource_path('views/welcome.blade.php'))) {
+            unlink(resource_path('views/welcome.blade.php'));
+        }
+
+        $this->runCommands(['npm install', 'npm run build']);
+    }
+
+    protected static function updateNodePackages(callable $callback, $dev = true): void
+    {
+        if (! file_exists(base_path('package.json'))) {
+            return;
+        }
+
+        $configurationKey = $dev ? 'devDependencies' : 'dependencies';
+
+        $packages = json_decode(file_get_contents(base_path('package.json')), true);
+
+        $packages[$configurationKey] = $callback(
+            array_key_exists($configurationKey, $packages) ? $packages[$configurationKey] : [],
+            $configurationKey
+        );
+
+        ksort($packages[$configurationKey]);
+
+        file_put_contents(
+            base_path('package.json'),
+            json_encode($packages, JSON_UNESCAPED_SLASHES | JSON_PRETTY_PRINT) . PHP_EOL
+        );
+    }
+
+    protected function runCommands($commands): void
+    {
+        $process = Process::fromShellCommandline(implode(' && ', $commands), null, null, null, null);
+
+        if ('\\' !== DIRECTORY_SEPARATOR && file_exists('/dev/tty') && is_readable('/dev/tty')) {
+            try {
+                $process->setTty(true);
+            } catch (RuntimeException $e) {
+                $this->output->writeln('  <bg=yellow;fg=black> WARN </> ' . $e->getMessage() . PHP_EOL);
+            }
+        }
+
+        $process->run(function ($type, $line) {
+            $this->output->write('    ' . $line);
+        });
+    }
+
+    public static function copyStubs(
+        ?array $files = null,
+        bool $force = false,
+        ?\Closure $basePath = null
+    ): void {
+        $files = is_array($files)
+            ? $files
+            : [
+                'tailwind.config.js',
+                'postcss.config.js',
+                'vite.config.js',
+            ];
+        if (! $basePath) {
+            $basePath = fn ($path = '') => base_path($path);
+        }
+
+        foreach ($files as $file) {
+            if (file_exists($basePath('tailwind.config.js')) && ! $force) {
+                continue;
+            }
+            copy(__DIR__ . '/../../../stubs/tailwind/' . $file, $basePath($file));
+
+            //copy(__DIR__ . '/../stubs/tailwind/' . $file, __DIR__ . '/../' . $file);
+            $content = file_get_contents($basePath($file));
+            $content = str_replace(
+                '{{ relative_path }}',
+                substr(realpath(__DIR__ . '/../../../'), strlen($basePath()), 999),
```

Comment:
why set length to 999?
`
If length is omitted or null, the substring starting from offset until the end of the string will be returned.
`